papermill_origami/noteable_dagstermill/ops.py

```diff
@@ -174,6 +169,8 @@ def _t_fn(step_context, inputs):
                 write_ipynb(nb_no_parameters, parameterized_notebook_path)
 
                 try:
+                    # create a dictionary that the engine can put data in so we can access it
+                    engine_out_data: Dict[str, Any] = {}
```

Comment:
This will tie in with the changes in the asset PR to allow us to publish the executed notebook URL as metadata